xiilib/_gunicorn/charm.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python3
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Flask Charm service."""
```

Comment:
Is this docstring accurate? This seems to be a module for gunicorn?

Reply:
Updated, thanks!